common/src/OpenSpaceNet.cpp

```diff
@@ -623,6 +640,13 @@ FileFeatureSink::Ptr OpenSpaceNet::initFeatureSink()
         definitions.emplace_back(FieldType::STRING, "catalog_id");
     }
 
+    for (int i = 0 ; i < args_.extraFields.size(); i+=2){
+        std::string fieldKey = args_.extraFields[i];
```

Comment:
Same comments as above wrt `move` and `auto`.

Reply:
RGR